README.md

```diff
@@ -124,65 +124,71 @@ practice to alternate morning/evening meetings once SIG has representatives
-| Rust: SDK | Every Tuesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4) | [#otel-rust](https://cloud-native.slack.com/archives/C03GDP0H023) and [Gitter](https://gitter.im/open-telemetry/opentelemetry-rust) | [calendar-rust](https://groups.google.com/a/opentelemetry.io/g/calendar-rust) | [Ted Young](https://github.com/tedsuo) |
-| Swift: SDK | Every Thursday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1LugL8r4bAkbTxZ1Gq_6j_8SDv1LuZaNeUdR-QLN4d48) | [#otel-swift](https://cloud-native.slack.com/archives/C01NCHR19SB) | [calendar-swift](https://groups.google.com/a/opentelemetry.io/g/calendar-swift) | [Alolita Sharma](https://github.com/alolita) |
-| eBPF | Every week on Tuesday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/13GK915hdDQ9sUYzUIWi4pOfJK68EE935ugutUgL3yOw) | [#otel-ebpf](https://cloud-native.slack.com/archives/C02AB15583A) | [calendar-ebpf](https://groups.google.com/a/opentelemetry.io/g/calendar-ebpf) | [Ted Young](https://github.com/tedsuo) |
-| Kubernetes Operator | Every other week on Thursday at 09:00 PT | [Google Doc](https://docs.google.com/document/d/1Unbs2qp_j5kp8FfL_lRH-ld7i5EOQpsq0I4djkOOSL4) | [#otel-operator](https://cloud-native.slack.com/archives/C033BJ8BASU) | [calendar-k8s-operator](https://groups.google.com/a/opentelemetry.io/g/calendar-k8s-operator) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
-| Community Demo Application | Every Wednesday at 8:00 PT | [Google Doc](https://docs.google.com/document/d/16f-JOjKzLgWxULRxY8TmpM_FjlI1sthvKurnqFz9x98) | [#otel-community-demo](https://cloud-native.slack.com/archives/C03B4CWV4DA) | [calendar-demo-app](https://groups.google.com/a/opentelemetry.io/g/calendar-demo-app) | [Austin Parker](https://github.com/austinlparker) |
+| Android: SDK + Automatic Instrumentation | Tuesday at 08:00 PT | [Google Doc](https://docs.google.com/document/d/1W72oElAJuYx3efM9wkH5l1iksdfwmTSkVl-H7KHTLnY) | [#otel-android](https://cloud-native.slack.com/archives/C05J0T9K27Q) | [calendar-android](https://groups.google.com/a/opentelemetry.io/g/calendar-android) | [Trask Stalnaker](https://github.com/trask) |
+| Collector | Wednesday alternating between 09:00 PT, 17:00 PT, 20:00 UTC+8 | [Google Doc](https://docs.google.com/document/d/1r2JC5MB7GupCE7N32EwGEXs9V_YIsPgoFiLP4VWVMkE) | [#otel-collector](https://cloud-native.slack.com/archives/C01N6P7KR6W) | [calendar-collector](https://groups.google.com/a/opentelemetry.io/g/calendar-collector) | [Juraci Paixão Kröhling](https://github.com/jpkrohling) |
```

Comment:
@svrnm I'm sorry, I missed this PR. Could we either have an exemption for the Collector SIG, or change the UTC rules?

The 20:00 UTC+8 time listed here is 05:00 Pacific time, 08:00 eastern time, and 14:00 Central European Time. So the only two timezones that attend are in Eastern North/South America and Europe since it is very late in APAC. If we pin this to UTC, the time goes to 07:00 Eastern time when daylight saving time ends, which will essentially make it a Europe-only meeting.

Could we either pin the current 17:00 PT meeting to UTC (making it 17:00 PDT and 16:00 PST, always 08:00 UTC+8) or pin all three to PT?

Reply:
@evan-bradley Can you put this into a separate issue such that it does not get lost and tag the other collector maintainers/approvers. I have no issue with that, as long as things stay fairly consistent and there is clarity around why a certain timezone was choosen

Reply:
Sure thing, thank you. I've opened an issue here: https://github.com/open-telemetry/community/issues/2264.